eng/Version.Details.xml

```diff
@@ -27,14 +27,14 @@
     </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="9.0.0-beta.23525.2">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="9.0.0-beta.23528.1">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>36d11a863c7ffc41e3730c806b0e98d32d1795ae</Sha>
+      <Sha>a4d49646dbac90883d3672f1c08134d84cca5707</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.XliffTasks" Version="1.0.0-beta.23523.1" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
+    <Dependency Name="Microsoft.DotNet.XliffTasks" Version="1.0.0-beta.23525.5" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
       <Uri>https://github.com/dotnet/xliff-tasks</Uri>
-      <Sha>83531fff1a39abe38de53e2e6b702bc1bb9c3bb4</Sha>
+      <Sha>0140d3d4a520d602a0721422eb656b78f16b5fba</Sha>
       <SourceBuild RepoName="xliff-tasks" ManagedOnly="true" />
```

Comment:
@jaredpar I don't have permissions in this repository. Please update this file and change the XliffTasks element like in https://github.com/dotnet/razor/commit/280095f357560f06c8d5e4d55b0ec62577a5a600#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3R102-R105.

That's because xliff-tasks got merged into arcade.